_episodes_rmd/01-raster-structure.Rmd

```diff
@@ -113,8 +118,19 @@ library(rgdal)
 # render DSM for tutorial content background
 DSM_HARV <- raster("data/NEON-DS-Airborne-Remote-Sensing/HARV/DSM/HARV_dsmCrop.tif")
 
-# code output here - DEM rendered on the screen
-plot(DSM_HARV, main = "Continuous Elevation Map\n NEON Harvard Forest Field Site")
+library(ggplot2)
```

Comment:
ideally it's good practice to load all libraries at the top of a script. this also allows students to see what packages are being used int he lesson. can we make this one small change @jsta @lachlandeer  ?  The rest of this looks good to me!